modules/ui/src/main/scala/lucuma/ui/sso/SSOConfig.scala

```diff
@@ -11,17 +11,16 @@ import io.circe.*
 import org.http4s.Uri
 import org.http4s.circe.*
 
+import java.time.Duration
 import java.util.concurrent.TimeUnit
 import scala.concurrent.duration.FiniteDuration
 
 case class SSOConfig(
-  uri:                        Uri,
-  readTimeoutSeconds:         Long = 3,
-  refreshTimeoutDeltaSeconds: Long = 10, // time before expiration to renew
-  refreshIntervalFactor:      Long = 1
```

Comment:
`refreshIntervalFactor` determined the period to refresh the token by dividing the time until next expiration by this. Since we are checking every second now, this is not needed anymore.